runtime/doc/diagnostic.txt

```diff
@@ -53,6 +53,9 @@ indicated with (+):
     code: The diagnostic code
     user_data: Arbitrary data plugins or users can add
 
+In the return value of |vim.diagnostic.get()|, keys `bufnr`, `end_lnum`,
```

Comment:
The wording here is ok, but I would put this in the docstring of vim.diagnostic.get directly (not that this requires modifying the comment in diagnostic.lua, then regenerating the docs with the gen_vimdoc.py script).

Reply:
The text is long and there are not enough line breaks.

```diff
diff --git a/runtime/doc/diagnostic.txt b/runtime/doc/diagnostic.txt
index b12a506..b6596ec 100644
--- a/runtime/doc/diagnostic.txt
+++ b/runtime/doc/diagnostic.txt
@@ -53,9 +53,6 @@ indicated with (+):
     code: The diagnostic code
     user_data: Arbitrary data plugins or users can add

-In the return value of |vim.diagnostic.get()|, keys `bufnr`, `end_lnum`,
-`end_col`, and `severity` are guaranteed to be present.
-
 Diagnostics use the same indexing as the rest of the Nvim API (i.e. 0-based
 rows and columns). |api-indexing|

@@ -481,7 +478,8 @@ get({bufnr}, {opts})                                    *vim.diagnostic.get()*
                  • severity: See |diagnostic-severity|.

     Return: ~
-        Diagnostic [] table A list of diagnostic items |diagnostic-structure|.
+        Diagnostic [] table A list of diagnostic items |diagnostic-structure|. In the
+        return value of |vim.diagnostic.get()|, keys `bufnr` , `end_lnum` , `end_col` , and `severity` are guaranteed to be present.

 get_namespace({namespace})                    *vim.diagnostic.get_namespace()*
     Get namespace metadata.
diff --git a/runtime/lua/vim/diagnostic.lua b/runtime/lua/vim/diagnostic.lua
index 0d1d01b..4cfa0f6 100644
--- a/runtime/lua/vim/diagnostic.lua
+++ b/runtime/lua/vim/diagnostic.lua
@@ -767,7 +767,7 @@ end
 ---                        - namespace: (number) Limit diagnostics to the given namespace.
 ---                        - lnum: (number) Limit diagnostics to the given line number.
 ---                        - severity: See |diagnostic-severity|.
----@return Diagnostic[] table A list of diagnostic items |diagnostic-structure|.
+---@return Diagnostic[] table A list of diagnostic items |diagnostic-structure|. In the return value of |vim.diagnostic.get()|, keys `bufnr`, `end_lnum`, `end_col`, and `severity` are guaranteed to be present.
 function M.get(bufnr, opts)
   vim.validate({
     bufnr = { bufnr, 'n', true },
```

Reply:
Remove this change and then it looks good to me.

Reply:
still waiting for this to be removed, before merging